designsafe/apps/data/models/taxonomy.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from django.db import models
+from django.contrib.auth.models import User
+from django.contrib.auth.validators import ASCIIUsernameValidator
+
+
+
+
+class CustomUser(User):
+    username_validator = ASCIIUsernameValidator()
+
+class Vocab(models.Model):
+    Term = models.CharField(max_length=40, unique=True)
+    Definition = models.TextField(max_length=200)
+    Source_of_Definition = models.UrlField(max_length=1000, unique=True)
+    Date = DateTimeField(default=datetime.now(), auto_now_add=True)
+    VocabBuilder = CustomUser()
```

Comment:
If this is to bind a user to a vocab record then use `ForeignKey`

Reply:
@xirdneh use ForeignKey for VocabBuilder? Or just for Vocab in general? The VocabBuilder feature is to list the individual that created the vocabulary word in the system.

Reply:
For `VocabBuilder` Something like:
```suggestion
    VocabBuilder = models.ForeignKey(settings.AUTH_USER_MODEL)
```
You might want to check the previous syntax I just typed it by memory and I usually make mistakes :)
You can check the [docs](https://docs.djangoproject.com/en/1.10/ref/models/fields/#django.db.models.ForeignKey) for more info.
Also, we're already doing something like that in other [models](https://github.com/DesignSafe-CI/portal/blob/master/designsafe/apps/auth/models.py#L20).

Reply:
Gotcha, that's what I was thinking, I just wanted to make sure. Thanks for your help!